docs/source/scheduling-state.rst

```diff
@@ -310,5 +310,6 @@ API
 
 .. autoclass:: Scheduler
    :members:
+   :inherited-members:
```

Comment:
I added this to get `SchedulerState.decide_worker`, but maybe we don't want all the Server, etc. methods showing up, and I should add `decide_worker` explicitly?